README.md

```diff
@@ -20,9 +20,10 @@ This software package is designed for preparing data that can be used to train g
     - [Meta in context learning](#meta-in-context-learning)
 
 ## Installation
-```git clone https://github.com/sambanova/generative_data_prep.git
-cd to the repo
-pip install -r requirements.txt
+```
+git clone https://github.com/sambanova/generative_data_prep.git
+cd generative_data_prep
+pip install .
```

Comment:
Incredible! Can we also upload to MyPi so people can pip install from there?

Reply:
yup, the CI flow will publish to pypi repository.

Reply:
👍